Userland/Services/WindowServer/Window.h

```diff
@@ -146,6 +146,10 @@ class Window final : public Core::Object
     float opacity() const { return m_opacity; }
     void set_opacity(float);
 
+    void disable_hit_tests(bool value)
```

Comment:
Please use `set_` prefix for setters and prefer addressing the positive (enabled) state: `set_hit_testing_enabled(bool)`, alternatively `set_hit_testing_disabled()` (with no way of going back)